vendor/github.com/jacobsa/gcloud/gcs/bucket.go

```diff
@@ -15,12 +15,15 @@
 package gcs
 
 import (
+	"crypto/md5"
```

Comment:
We shouldn't make changes to vendor folder directly. We are refering them by version  in go.mod file. Any changes you need to make to vendor code should be made in their respective repositories and version has to be updated gcsfuse go.mod file. 
Similar to how you make bucket doesn't exist fix:
Code changes in gcloud: https://github.com/jacobsa/gcloud/pull/28
version update in gcsfuse: https://github.com/GoogleCloudPlatform/gcsfuse/pull/796

Irrespective of that, for go storage library integration we should avoid adding code in jacobsa/gcloud library. All this effort is to remove dependency on jacobsa/gcloud, adding new functions there is like adding more dependency.

Reply:
Should I add this function in bucket_handle file only?

Reply:
Should I add these changes in bucket_handle.go file only?

Reply:
yes. please create a storage/storageutil/object_attrs.go

Reply:
fixed